packages/login-ui/redux/thunks.js

```diff
@@ -103,11 +104,20 @@ export const onSubmitLoginForm = (password) => async (dispatch, getState) => {
   const { selectedWallet, lastLoginNetworkId } = state.get(DUCK_PERSIST_ACCOUNT)
   const accountWallet = new AccountEntryModel(selectedWallet)
   const { network } = getCurrentNetworkSelector(state)
-  const { selectedNetworkId } = getState().get(DUCK_NETWORK)
+  const { selectedNetworkId, selectedProviderId } = getState().get(DUCK_NETWORK)
   if (lastLoginNetworkId !== selectedNetworkId) {
     dispatch(PersistAccountActions.clearWalletsAddressCache())
   }
-  dispatch(PersistAccountActions.updateLastNetworkId(selectedNetworkId))
+  const availableNetwork = await checkNetwork(selectedNetworkId, selectedProviderId)
+  if(availableNetwork){
+    if(availableNetwork.selectedNetworkId !== selectedNetworkId) dispatch(NetworkActions.networkSetNetwork(availableNetwork.selectedNetworkId))
```

Comment:
Same moment with space after if and better avoid one line if. Let's add curl braces 
```
if () {
 ... 
}
```